torch/distributed/rpc/api.py

```diff
@@ -527,11 +527,21 @@ def _invoke_rpc(to, func, rpc_type, args=None, kwargs=None, rpc_timeout=UNSET_RP
 
         is_async_fn = hasattr(func, "_wrapped_async_rpc_function")
 
+        if is_async_fn:
+            wrapped = func._wrapped_async_rpc_function
+            if isinstance(wrapped, torch.jit.ScriptFunction):
+                func = wrapped
```

Comment:
This wouldn't work if we reverse the order of the decorators (jit.script first and then async_function)? I'm not sure if its possible to make this work such that the user doesn't have to depend on an order (ex: not sure if we can script a method that already has a decorator).

Hopefully we can make this order agnostic and add an appropriate unit test verifying this. If not, can we add this limitation to the documentation of `async_function`?

Reply:
Good point, will me test and document.